link_options.go

```diff
@@ -63,6 +63,26 @@ type SenderOptions struct {
 
 	// SourceAddress specifies the source address for this sender.
 	SourceAddress string
+
+	// TargetCapabilities is the list of extension capabilities the sender desires.
+	TargetCapabilities []string
+
+	// TargetDurability indicates what state of the peer will be retained durably.
+	//
+	// Default: DurabilityNone.
+	TargetDurability Durability
+
+	// TargetExpiryPolicy determines when the expiry timer of the peer starts counting
+	// down from the timeout value.  If the link is subsequently re-attached before
+	// the timeout is reached, the count down is aborted.
+	//
+	// Default: ExpirySessionEnd.
+	TargetExpiryPolicy ExpiryPolicy
+
+	// ExpiryTimeout is the duration in seconds that the peer will be retained.
+	//
+	// Default: 0.
+	TargetExpiryTimeout uint32
```

Comment:
uint32 => time.Duration? We do that with BatchMaxAge, but perhaps it hasn't been consistent, or we have a rule to follow here.

Reply:
This is expressed in seconds, will update the docs accordingly.

Reply:
Oh wait, the docs already state that this is expressed in seconds.

BatchMaxAge is a client-side construct and provides for higher fidelity.

Generally speaking, I don't think we have a hard-and-fast rule for this in the guidelines.  That said, the pattern that we've been implicitly following is to use a `time.Duration` when sub-second precision is available, else use an int/uint to describe the number of units.